extensions/wikia/AdEngine/js/spec/lookup/rubiconTargeting.spec.js

```diff
@@ -0,0 +1,70 @@
+/*global beforeEach, describe, it, modules, expect, spyOn*/
+describe('ext.wikia.adEngine.lookup.rubiconTargeting', function () {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)